client/src/assets/scss/styles.scss

```diff
@@ -0,0 +1,379 @@
+@import './_variables.scss';
+@import 'node_modules/bootstrap/scss/bootstrap.scss';
```

Comment:
Bootstrap and Material may be some sort of overkill. Why not stick to one library/framework

Reply:
Thank you for the feedback. The app now uses CDN instead